src/shared/ui/GenreSelect/GenerMockData.js

```diff
@@ -0,0 +1,3 @@
+import react from "react"
```

Comment:
do we really need to import `react` here?

Reply:
removed